akka-stream/src/main/resources/reference.conf

```diff
@@ -40,6 +40,17 @@ akka {
 
       # Maximum number of elements emitted in batch if downstream signals large demand
       output-burst-limit = 1000
+
+      interpreter {
```

Comment:
Isn't "interpreter" an implementation detail/name?
I.e. is this a good name for a section in the official configuration.